credsweeper/rules/config.yaml

```diff
@@ -893,6 +893,17 @@
     - src
     - doc
 
+- name: Gitlab Incoming Email Token
+  severity: info
+  type: pattern
+  values:
+    - (^|[^0-9A-Za-z._/+-])(?P<value>[a-z0-9]{24,25})([^=0-9A-Za-z+/_-]|$)
```

Comment:
min_line_len: 25
but {24,25} in regex
Doesn't affect the result I think, but looks weird.